### ISSUE TITLE
webDriverProxy strips grid hub port number

### ISSUE THREAD

**Issue by gregjhogan** [angular/protractor#3051]
When I specify a web driver proxy, the requests to the hub are made without the hub port number.

You can reproduce the issue by following the example test [here](https://angular.github.io/protractor/#/) and using the following config:

``` javascript
exports.config = {
  seleniumAddress: 'http://{selenium-hub-host}:4444/wd/hub',
  webDriverProxy: 'http://127.0.0.1:8888',
  capabilities: {
    browserName: 'chrome'
  },
  specs: ['*.spec.js']
};
```

Fire up fiddler (it runs on 127.0.0.1:8888 by default), run the test, and the test times out obtaining a session from the hub because the initial request is made to the hub on the wrong port (notice port 4444 is not specified):

```
POST http://{selenium-hub-host}/wd/hub/session HTTP/1.1
Accept: application/json; charset=utf-8
Content-Length: 58
Content-Type: application/json;charset=UTF-8
Host: {selenium-hub-host}
Connection: close

{"desiredCapabilities":{"browserName":"chrome","count":1}}
```
- Node Version: `5.3.0`
- Protractor Version: `3.2.1`
- Operating System and Version `Windows 10`

**Comment by juliemr** [comment]
I'm also seeing this issue with the latest version of Protractor. Probably an issue with selenium-webdriver being used incorrectly, but I need to investigate more.

**Comment by juliemr** [comment]
This was an issue with selenium-webdriver 2.53 and is fixed in 3.0, which is used in our beta branch.
